dist/lib/rules/tooltip-not-recommended.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Q: will we be able to remove all the source mapping files once they have been converted to TypeScript?

Reply:
we should be able to